docs/api.md

```diff
@@ -198,6 +200,7 @@
   * [page.waitForSelector(selector[, options])](#pagewaitforselectorselector-options)
   * [page.waitForTimeout(milliseconds)](#pagewaitfortimeoutmilliseconds)
   * [page.waitForXPath(xpath[, options])](#pagewaitforxpathxpath-options)
+  * [page.webvitals](#pagewebvitals)
```

Comment:
This casing is inconsistent with the other API names used. To fix, use `webVitals`